From dd072e8e639554298b9d1205540cc4555efef1dd Mon Sep 17 00:00:00 2001
From: Ramya Gnanasekar <quic_rgnanase@quicinc.com>
Date: Fri, 4 Nov 2022 00:22:17 +0530
Subject: [PATCH] iw: print handler to display 320MHz bitrate

This patch adds handler to print tx/rx bitrates
for 320MHz

Signed-off-by: Ramya Gnanasekar <quic_rgnanase@quicinc.com>

diff --git a/station.c b/station.c
index 970e8a5..5618200 100644
--- a/station.c
+++ b/station.c
@@ -239,6 +239,8 @@ void parse_bitrate(struct nlattr *bitrate_attr, char *buf, int buflen)
 		pos += snprintf(pos, buflen - (pos - buf), " 80P80MHz");
 	if (rinfo[NL80211_RATE_INFO_160_MHZ_WIDTH])
 		pos += snprintf(pos, buflen - (pos - buf), " 160MHz");
+	if (rinfo[NL80211_RATE_INFO_320_MHZ_WIDTH])
+		pos += snprintf(pos, buflen - (pos - buf), " 320MHz");
 	if (rinfo[NL80211_RATE_INFO_SHORT_GI])
 		pos += snprintf(pos, buflen - (pos - buf), " short GI");
 	if (rinfo[NL80211_RATE_INFO_VHT_NSS])
-- 
2.17.1

